pyqt-apps/siriushla/widgets/pvnames_tree.py

```diff
@@ -84,27 +85,66 @@ def items(self, value):
         self._items = value
         self._add_items()
 
+    def check_all(self):
+        """Check all items."""
+        for i in range(self.topLevelItemCount()):
+            self.topLevelItem(i).setCheckState(0, Qt.Checked)
+
+    def uncheck_all(self):
+        """Uncheck all items."""
+        for i in range(self.topLevelItemCount()):
+            self.topLevelItem(i).setCheckState(0, Qt.Unchecked)
+
     def _add_item(self, item):
 
         if isinstance(item, str):
-            row = (SiriusPVName(item), )
+            pvname = item
+            row = [item, ]
         else:
+            if not isinstance(item[0], str):
+                raise ValueError
+            pvname = item[0]
             row = [item[0], ]
             row.extend([str(i) for i in item[1:]])
 
-        key = row[0] if isinstance(row[0], SiriusPVName) \
-            else SiriusPVName(row[0])
         pvals = []
+        if re.match('^.*-.*:.*-.*:.*-.*$', pvname):
```

Comment:
@gu1lhermelp, wouldn't a `try... except` around the `SiriusPVName` call be more suitable here, instead of the `regexp` checking?